contrib/container-fedora-25/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.fedoraproject.org/fedora:25
+
+RUN cd /etc/yum.repos.d && curl -O https://copr.fedorainfracloud.org/coprs/adelton/custodia/repo/fedora-25/adelton-custodia-fedora-25.repo
```

Comment:
Do we have some more official copr repo of reasonably recent custodia version? The latest build in koji as of today is custodia-0.1.0-6.fc25.